java/org/apache/coyote/AbstractProtocol.java

```diff
@@ -351,12 +351,15 @@ public String getName() {
 
     private String getNameInternal() {
         StringBuilder name = new StringBuilder(getNamePrefix());
-        name.append('-');
         if (getPath() != null) {
-            name.append(getPath().getFileName().toString());
+            for (Path path: getPath()) {
```

Comment:
Any reason not to use the path as-is? Will this name be used as file name?

Reply:
This is the name of the connector, for example:

```
30-Nov-2020 20:33:08.007 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-nio-8080"]
30-Nov-2020 20:33:08.015 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-apr-tmp-protected-tomcat.socket"]
```

The name "http-apr-/tmp/protected/tomcat.socket" seems ugly to me.

Reply:
While it seems ugly, it makes it perfectly clear that it is a path on the local filesystem, doesn't it?

Reply:
I've updated it, now looks like this:

```
30-Nov-2020 23:05:45.172 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-nio-8080"]
30-Nov-2020 23:05:45.180 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-apr-/tmp/protected/tomcat.socket"]
```

Is this all ok?

Reply:
Absolutely, it is now crystal clear that that is a domain socket.

Reply:
Path used as is.